### PR TITLE
Add HTTP endpoint to render ReportQueries

### PR DESCRIPTION
**Pull request by chancez**
```
export REPORT_API_URL=127.0.0.1:8080
$ curl $REPORT_API_URL/api/v2/reportqueries/metering-chancez3/namespace-memory-request/render --data '{"inputs": [], "start": "2019-09-01T00:00:00-00:00", "end": "2019-09-30T00:00:00-00:00"}' -XPOST -H "Content-Type: application/json"

SELECT
  timestamp '2019-09-01 00:00:00.000' AS period_start,
  timestamp '2019-09-30 00:00:00.000' AS period_end,
  namespace,
  sum(pod_request_memory_byte_seconds) as pod_request_memory_byte_seconds
FROM hive.metering.datasource_metering_chancez3_pod_memory_request_raw
WHERE "timestamp" >= timestamp '2019-09-01 00:00:00.000'
AND "timestamp" < timestamp '2019-09-30 00:00:00.000'
AND dt >= '2019-09-01'
AND dt <= '2019-09-30'
GROUP BY namespace
```

### CODE REVIEW

File: pkg/operator/http.go [hunk 1]
```diff
@@ -36,8 +38,9 @@ var ErrReportIsRunning = errors.New("the report is still running")
 var prometheusMiddleware = chiprometheus.NewMiddleware("reporting-operator")
 
 const (
-	APIV1ReportsGetEndpoint    = "/api/v1/reports/get"
-	APIV2ReportsEndpointPrefix = "/api/v2/reports"
+	APIV1ReportsGetEndpoint        = "/api/v1/reports/get"
+	APIV2ReportEndpointPrefix      = "/api/v2/reports"
+	APIV2ReportQueryEndpointPrefix = "/api/v2/reportqueries"
 )
```
Comment (EmilyM1) [inline pkg/operator/http.go]: Inconsistent in singular/plural. Line 41 - Reports, line 42,43 singular report

**Comment by chancez** [comment]
@EmilyM1 @timflannagan1 fixed

**Pull request merged**